dist/index.html

```diff
@@ -7,18 +7,34 @@
   <title>Travel Tracker</title>
 </head>
 <body>
-  <main class="main">
-    <header class="header">
-      <div class="container">
-        <div class="header-logo">TRAVEL TRACKER</div>
-        <div class="header-menu">
-          <a class="link1">another HTML page</a>
-          <a class="link2">another HTML page</a>
-          <a class="link3">and another</a>
-        </div>
-      </div>
-    </header>
+  <header class="header">
+    <section class="header-container">
+      <h1 class="header-title">TRAVEL TRACKER</h1>
+      <section class="header-right">
+        <h2 class="welcome-title">Welcome, User</h2>
+        <h3 class="total-spent">You've spent $$$</h2>
+      </section>
+    </section>
+  </header>
+  <main>
+    <!-- <aside class="aside">
```

Comment:
I like your idea of having the place to plan a new trip in a very visible aside! I think that'll make it easier on the user to plan in fewer clicks